src/components/LocationBias.tsx

```diff
@@ -55,13 +56,15 @@ export function LocationBias({
   const locationBias = useAnswersState(s => s.location.locationBias);
   const [isFetchingLocation, setIsFetchingLocation] = useState<boolean>(false);
   const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses);
+  const loadingIndicatorCss = twMerge(cssClasses.loadingIndicatorContainer, (!isFetchingLocation && 'invisible'));
+  const sourceCss = twMerge(cssClasses.source, 'whitespace-pre');
```

Comment:
can we add the class directly to `source` without using `twMerge` since it's not conditionally added?